admin_tools/menu/tests.py

```diff
@@ -19,6 +22,41 @@ def test_custommenu(self):
             pass
 
 
+class DeleteBookMarkTest(TestCase):
+    fixtures = ['users.json']
+
+    def setUp(self):
+        self.superuser_credentials = ('superuser', '123')
+        self.staff_credentials = ('staff', '123')
+        user = User.objects.get(username=self.superuser_credentials[0])
+        self.bookmark = Bookmark.objects.create(
+            user=user,
+            url='/test/',
+            title='test'
+        )
+        self.delete_bookmark_url = reverse('admin-tools-menu-remove-bookmark', args=(self.bookmark.id,))
+
+    def _login_user(self, username, password):
+        try:
+            user = User.objects.get(username=username)
+            self.client.force_login(user)
+        except AttributeError:  # in Django<1.9
+            logged_in = self.client.login(username=user.username, password=password)
```

Comment:
You can directly use `username=username`. At first I was confused how you are using `user.username` if `user` is not passed to the function and also `user` is not fetched before `try` block.

Also I'm wondering what is the benefit of this function at all. With `force_login` you have the convenience to log in a user without knowing it's password. In this case you still need to call `_login_user` with username and password in order to fallback for Django < 1.9.